fastlane/Fastfile

```diff
@@ -150,6 +150,13 @@ lane :update_hybrid_common do |options|
     open_pr: options[:open_pr] || false,
     automatic_release: options[:automatic_release] || false
   )
+
+  # Update `yarn.lock` too
+  sh('yarn install --no-immutable')
+
+  sh("git", "add", "yarn.lock")
+  sh("git", "commit", "-m", "Update yarn.lock")
+  sh("git", "push")
```

Comment:
This should work, but we do have a couple of fastlane commands to help with these:
```
commit_current_changes(commit_message: 'Update yarn.lock')
push_to_git_remote(set_upstream: true)
```
Note that will commit all changes, not only changes to the yarn.lock. We do use these commands in the `bump` fastlane action.

Reply:
Oh thanks! Updating